src/bridge/command.rs

```diff
@@ -176,8 +181,8 @@ fn nvim_cmd_impl(bin: &str, args: &[String]) -> TokioCommand {
     }
 }
 
-fn build_nvim_cmd_with_args(bin: &str) -> TokioCommand {
-    let mut args = vec!["--embed".to_string()];
+fn build_nvim_cmd_with_args(bin: String, mut args: Vec<String>) -> TokioCommand {
```

Comment:
I changed `&[String]` to `Vec<String>` so that the `Vec` instance can be reused. It can remove one redundant clone here.